lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -31,20 +31,19 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       PROVIDER_SPECIFIC_FIELDS = {
         'ovirt' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"])
+          Fields::Field.new(:label => _('UUID'), :path => [:uuid])
```

Comment:
you're right, it's an [alias for ovirt](https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/ovirt.rb#L11) and [vmware](https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/vmware.rb#L415) so this should be most likely renamed to Datacenter in hammer for both

bottom line, the uuid on this should probably be datacenter (both label and path), other than that :+1: